websites/docs-smoke-test/src/content/index.mdx

```diff
@@ -61,18 +101,36 @@ This is a second **paragraph**
   </LearningPathCard>
   <ImageCard>
 
-  ![learning path](/content/files/logged-in-learning-path.svg)
+  ![learning path](/content/files/logged-out-learning-path.svg)
 
   </ImageCard>
 </Cards>
 
-# Logged in or logged out content
 
-<IfUserLoggedIn>
- ## Welcome back!
-</IfUserLoggedIn>
+<div style="display:flex;">
+  <PrimaryButton onClick={() => window.location.href = 'https://commercetools.eu.auth0.com/authorize?response_type=code&client_id=xLk8EDUCc8PKqCbrSJCnuahvn86nEn4z&redirect_uri=https://docs.commercetools.com&scope=openid%20profile'}>
```

Comment:
it could be something like this (did not test it)

```suggestion
  <PrimaryButton onClick={() => window.location.href = `https://auth.id.commercetools.com/authorize?response_type=code&client_id=xLk8EDUCc8PKqCbrSJCnuahvn86nEn4z&redirect_uri=${window.location.origin}&scope=openid%20profile`}>
```

Reply:
@gabriele-ct